app/Console/Commands/MigrateSettingsData.php

```diff
@@ -33,6 +34,11 @@ class MigrateSettingsData extends Command implements Isolatable
             'handler' => 'enableEngagementNotificationsMigration',
             'description' => 'Replaces older format of notifications_settings with only ["engagements" => "1"]. Setting the engagement notifications on be default. If the notifications_settings contains a valid engagements setting, then no changes are made.',
         ],
+        'SchemalessPrompts' => [
+            'version' => '1.8.0',
+            'handler' => 'schemalessPromptsMigration',
+            'description' => 'Moves users’ dismissed_customize_prompt_at value to the prompts schemaless attributes column.',
```

Comment:
Now that this handles changes for Orgs and Fros the description may need to be updated